src/current/cockroachcloud/create-your-cluster.md

```diff
@@ -28,21 +28,17 @@ CockroachDB {{ site.data.products.dedicated }} advanced clusters have access to
 
 ## Step 2. Select the cloud provider
 
-In the **Cloud provider** section, select your deployment environment: **Google Cloud**, **AWS**, or **Microsoft Azure** (Limited Access).
+In the **Cloud provider** section, select your deployment environment: **Google Cloud**, **AWS**, or **Microsoft Azure**.
```

Comment:
Suggest mentioning here or in the step right above that there's no azure option for dedicated advanced